packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -100,7 +100,7 @@ export default function MemberListItem(props: Props) {
               <div className="mb-1 flex">
                 <span className="mr-1 text-sm font-bold leading-4">{name}</span>
 
-                {props.member.isMissingSeat && <TeamPill color="red" text={t("hidden")} />}
+                {/* {props.member.isMissingSeat && <TeamPill color="red" text={t("hidden")} />} */}
```

Comment:
We don't need this for V2 right?

Reply:
We do until team billing is properly implemented

Reply:
add context note